backend/src/routes/v0/tool/info.py

```
@@ -0,0 +1,77 @@
+    },
+)
+async def get_a2a_agent_card(
+    config: dict[str, A2AServer] = Body(openapi_examples=A2A_DICT_EXAMPLE),
+):
+    try:
+        agent_cards = tool_service.agent_cards(config)
+
+        return JSONResponse(
+            content={"agent_cards": agent_cards}, status_code=status.HTTP_200_OK
+        )
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)